### PR TITLE
Translate instana-version-resource pipeline to Tekton

### PR DESCRIPTION
**Pull request by Siddhant-Golangade**
#Why
Translate instana-version-resource pipeline to Tekton

#What
Converted configuration from Concourse to Tekton

### CODE REVIEW

**Comment by bennett-white** [comment]
So we have two options to handle the docker build here.

1. Switch to a build tool such as podman or buildah that does not require a docker daemon to be running. Since there is no need to run a docker container in this build, this would be my suggested option. This would involve finding an image with podman/buildah available. The redhat podman image is one that I typically use. 

2. Set up the docker daemon for use within each task by following the example here https://github.com/tektoncd/pipeline/blob/main/examples/v1/taskruns/dind-sidecar.yaml

File: tekton-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          value: $(params.branch)
+        - name: deleteExisting
+          value: "true"
+
+    - name: build-instana-version-resource-image
+      taskSpec:
+        params:
+          - name: cloned-dir
+            type: string
+            value: instana-version-resource
```
Comment (bennett-white) [inline tekton-pipeline.yaml:38]: Should be default not value

Comment (bennett-white) [inline tekton-pipeline.yaml]: I think this should be `default` and not `value`

Comment (Siddhant-Golangade) [reply tekton-pipeline.yaml]: Fixed!

**Review by bennett-white — APPROVED** [review]
LGTM

**Pull request merged**